### DISCUSSION TITLE
hakoniwa-single_robotの動作確認（Windows 11、WSL2、Ubuntu 22.04）

### DISCUSSION THREAD

**Discussion by YusukeKato** [toppers/hakoniwa#166] [General]
すでに確認されているかもしれませんが、下記の環境にてhakoniwa-single_robotの動作確認ができたので一応報告です。

- Windows 11 Home 22H2
- WSL2 / Ubuntu 22.04.3 LTS

こちらのリポジトリを使用させていただきました（v1.4.0）。
https://github.com/toppers/hakoniwa-single_robot

**Comment by takasehideki** [comment]
@YusukeKato お試しいただき && 報告をありがとうございます！
hakoniwa-single_robot は開発収束気味というところが実情でして，当方ではまだ明確に試していませんでした．機を見て README 更新したいと思います！

**Comment by YusukeKato** [comment]
すみません、こちらもオープンのままになっていたので、クローズさせていただきます。
ご対応ありがとうございました。
